Dockerfile

```diff
@@ -55,14 +49,12 @@ RUN curl -O ftp://ftp.unidata.ucar.edu/pub/netcdf/netcdf-4.4.1.1.tar.gz \
     && tar xzf netcdf-4.4.1.1.tar.gz \
     && cd netcdf-4.4.1.1 \
     && /bin/sh configure --prefix=/usr/local \
-    && make \
+    && make -j \
     && make install \
     && cd .. \
     && rm -rf netcdf-4.4.1.1 \
     && rm -rf netcdf-4.4.1.1.tar.gz
 
-RUN yum -y install httpd
-RUN yum clean all
 RUN systemctl enable httpd.service
```

Comment:
Didn't see this in here previously.  This is probably effectively a no-op: Apache isn't installed, and even if it was, SystemD wouldn't be running as PID 1 (or at all) since there's a startup script used instead.